examples/bastion/config_no_pub_ip.json

```diff
@@ -4,15 +4,18 @@
     "install_from": "",
     "admin_user": "hpcadmin",
     "variables": {
-        "jumpbox_image": "OpenLogic:CentOS:7.7:latest",
+        "jumpbox_image": "OpenLogic:CentOS-CI:7-CI:latest",
```

Comment:
This image should not be used - see details here: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/using-cloud-init#centos